build.gradle

```diff
@@ -427,10 +427,9 @@ dependencyManagement {
       entry 'spring-messaging'
     }
 
-  //CVE-2022-25857
-  dependencySet(group: 'org.yaml', version: '1.31') {
+    dependencySet(group: 'org.yaml', version: '1.31') {
```

Comment:
there is no increment of snakeyaml it just looks like spacing/formatting changed here?

Reply:
Yes. Just aligning it with the entries above.